### PR TITLE
add 'add to world' button

### PR DESCRIPTION
**Pull request by treeplate**

### CODE REVIEW

File: application/lib/editor.dart [hunk 1]
```diff
@@ -63,35 +64,64 @@ class _EditorState extends State<Editor> {
...
-      case 'opened': return 'Opened?';
-      case 'pattern': return 'Pattern';
-      case 'pileClass': return 'Pile class';
-      case 'position': return 'Position';
-      case 'size': return 'Size';
-      case 'surface': return 'Surface';
-      case 'underDescription': return 'Description (under)';
-      case 'writing': return 'Writing';
-      default: return property;
+      case 'backDescription':
```
Comment (Hixie) [inline application/lib/editor.dart:67]: let's not do this...

File: application/lib/editor.dart [hunk 2]
```diff
@@ -105,66 +135,104 @@ class _EditorState extends State<Editor> {
           key: ValueKey<String>(property),
           label: _prettyName(property),
           rootClass: parts[1],
-          value: widget.atom.ensurePropertyIs<AtomPropertyValue>(property)?.value,
+          value:
+              widget.atom.ensurePropertyIs<AtomPropertyValue>(property)?.value,
```
Comment (Hixie) [inline application/lib/editor.dart:139]: or this

**Comment by Hixie** [comment]
there's a ton of formatting changes here that make things less readable...

**Comment by treeplate** [comment]
Branch https://github.com/treeplate/cuddlyworld-ide/tree/add2world2 is better

**Pull request closed without merging**